Marlin/src/pins/sanguino/pins_SANGUINOLOLU_11.h

```diff
@@ -166,9 +166,15 @@
       #define LCD_PINS_ENABLE                 16
       #define LCD_PINS_D4                     11
 
-      #define BOARD_ST7920_DELAY_1 DELAY_NS(0)
-      #define BOARD_ST7920_DELAY_2 DELAY_NS(188)
-      #define BOARD_ST7920_DELAY_3 DELAY_NS(0)
+      #ifndef BOARD_ST7920_DELAY_1
+        #define BOARD_ST7920_DELAY_1 DELAY_NS(0)
+      #endif
+      #ifndef BOARD_ST7920_DELAY_2
+        #define BOARD_ST7920_DELAY_2 DELAY_NS(188)
+      #endif
+      #ifndef BOARD_ST7920_DELAY_3
+        #define BOARD_ST7920_DELAY_3 DELAY_NS(0)
+      #endif
```

Comment:
Since this file is the one that others want to include, `#ifndef` is added to allow the parent files to override.

Reply:
No.
This would only make sense if you'd know: the here introduced delays are better than the already defined values from the CPU-delays for all sanguino -boards where the value is not explicitly defined.

https://github.com/MarlinFirmware/Marlin/blob/2.0.x/Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.h#L45-L76

https://github.com/MarlinFirmware/Marlin/blob/2.0.x/Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.cpp#L36-L44

But you can't know.

Reply:
There are three levels of overrides to wrangle, where USER overrides BOARD overrides CPU. The basic assumption was that the CPU's choice would be the main fallback, with overrides provided from some earlier point.

But, along the way it was decided to have overrides inside of pins files for certain display / board combinations. Originally these were only the regular `ST7920_DELAY_x` items and there were not many.

But, these have been multiplying, so every pins file then has to deal with override logic.

I decided to change the overrides inside of pins files to add another level of defaults and then put the override behavior for "USER overrides BOARD" in one place. I tried to do this while preserving the same outcomes, but I might have missed something in making those changes.

Reply:
You defined this for a whole group of boards. Before only one board of this group used this. Are you sure this is better for all of them.

Reply:
The problem was introduced in #18160. Where @git-user44 tried to fix his problems with a new board in 'pins_SANGUINOLOLU_11.h' instead of in an extra file for the new board. This caused trouble with the boards where the defines already existed. This PR originally tried to fix this by undefining before defining for the boards in trouble. As a result of my complain this PR was changed to introduce the new board and revert the change in 'pins_SANGUINOLOLU_11.h'. You reintroduced a improved version of the change in 'pins_SANGUINOLOLU_11.h' but now again changing the behavior of all boards have not had the defines before.

Reply:
With so many issues and comments to read each day, and so few users capable of submitting fixes, the workload is quite overwhelming and mistakes will be made. Thanks for pointing this out. With any luck I will be able to to fix it over the weekend.